locales/zhCN.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Let's hold off from adding any translations (even for testing). The first thing we should do is get all of the `enUS` into its json.